development/kops/set_environment.sh

```diff
@@ -24,10 +24,10 @@ export NODE_INSTANCE_TYPE=${NODE_INSTANCE_TYPE:-t3.medium}
 export NODE_ARCHITECTURE=${NODE_ARCHITECTURE:-amd64}
 export UBUNTU_RELEASE=${UBUNTU_RELEASE:-focal-20.04}
 export IPV6=${IPV6:-false}
-if [ "$RELEASE_BRANCH" == "1-29" ]; then
-	export KOPS_VERSION="1.29.0-alpha.3"
+if [ "$RELEASE_BRANCH" >= "1-30" ]; then
```

Comment:
We want this to be 29 or higher. Also not sure the string check will work that way. We want to strip the minor version off.

Reply:
Verified from the logs that RELEASE_BRANCH takes the string as 1.30
`++ export RELEASE_BRANCH=1-30
++ RELEASE_BRANCH=1-30`
modified the comparator check